fastlane/lib/fastlane/actions/copy_artifacts.rb

```diff
@@ -43,7 +43,7 @@ def self.run(params)
       #####################################################
 
       def self.description
-        "Small action to save your build artifacts. Useful when you use reset_git_repo"
+        "Copy and save your build artifacts (Useful when you use reset_git_repo)"
```

Comment:
capital `U` on useful

Reply:
Should I change it to lowercase?

Reply:
it's no big deal, but lower case `u` would be grammatically correct. You can edit in the github ui

Reply:
yep, I honestly just didn't know. That the problem of us non native speakers...